lib/SimpleTest/SimpleTestRunner.cpp

```diff
@@ -29,23 +31,23 @@ extern "C" int mull_simple_test_printf(const char *fmt, ...) {
   return 0;
 }
 
-class Mull_SimpleTest_Resolver : public JITSymbolResolver {
+class Mull_SimpleTest_Resolver : public RuntimeDyld::SymbolResolver {
 public:
 
-  JITSymbol findSymbol(const std::string &Name) {
+  RuntimeDyld::SymbolInfo findSymbol(const std::string &Name) {
     //if (Name == "_printf") {
     //  return findSymbol("mull_simple_test_printf");
     //}
 
     if (auto SymAddr = RTDyldMemoryManager::getSymbolAddressInProcess(Name)) {
-      return JITSymbol(SymAddr, JITSymbolFlags::Exported);
+      return RuntimeDyld::SymbolInfo(SymAddr, JITSymbolFlags::Exported);
     }
 
-    return JITSymbol(nullptr);
+    return RuntimeDyld::SymbolInfo(nullptr);
```

Comment:
You can use `typedef JITSymbol RuntimeDyld::SymbolInfo;`, after update we can just drop that line.